test-utils-base/src/commonMain/kotlin/ui_thread_testing.kt

```diff
@@ -15,7 +15,7 @@
 
  */
 
-package com.splendo.kaluga.test
```

Comment:
File name can be camelCase?

Reply:
If you're going to change it, the kotlin convention is `UpperCamel`: https://kotlinlang.org/docs/coding-conventions.html#source-file-names

but this is a convention I really dislike personally. You don't see any difference between files with many classes or just one (though the IDE does show this).

Reply:
resolving this as it's not directly related to this PR, please raise in kaluga meeting if you really think filenames should follow convention or want to discuss it in some way.